bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/BusinessPartnerService.kt

```diff
@@ -209,6 +223,24 @@ class BusinessPartnerService(
         }
 
 
+    private fun requestNewCleaning(orchestratorBusinessPartnersDto: List<BusinessPartnerGenericDto>): TaskCreateResponse {
+        return orchestrationApiClient.goldenRecordTasks.createTasks(
+            TaskCreateRequest(
+                TaskMode.UpdateFromSharingMember, orchestratorBusinessPartnersDto
+            )
+        )
+    }
+
+    private fun updateSharingState(entity: BusinessPartner, stateDto: TaskClientStateDto) {
+        sharingStateService.upsertSharingState(
+            SharingStateDto(
```

Comment:
In the update you set the taskId but not the sharingState, so it stays here initial. But in your logic it should be set to Pending instead.